website/content/docs/upgrading/version-guides/0.11.0.mdx

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+page_title: Version 0.11.0 - Upgrade Guide
+description: |-
+  This upgrade guide describes any considerations for upgrading to Waypoint 0.11.0.
+---
+
+# Upgrading to Waypoint 0.11.0 from 0.10.x
+
+Waypoint 0.10.x can be upgraded to Waypoint 0.11.0 using a [standard upgrade](/waypoint/docs/upgrading#standard-upgrade).
+There are some considerations depending on what platform your server is installed on.
+There are no breaking changes in the protocol or `waypoint.hcl` configuration file.
```

Comment:
Thank you for articulating this - I almost omitted this sentence in my draft, but since neither the protocol nor the config file have breaking changes, I did leave it and left the issue to be discussed below, but I like that this calls it out earlier.